dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/DateUtils.java

```diff
@@ -431,8 +492,7 @@ public static Date getEndOfHour(Date inputDay) {
      * @return current date
      */
     public static Date getCurrentDate() {
-        return DateUtils.parse(DateUtils.getCurrentTime(),
-            Constants.YYYY_MM_DD_HH_MM_SS);
+        return DateUtils.parse(DateUtils.getCurrentTime(), Constants.YYYY_MM_DD_HH_MM_SS, null);
```

Comment:
There is no need to use `DateUtils.parse`. The Date is not bind with `TimeZone`. So return new Date is OK.